src/elements/types/ObjectExpression.js

```diff
@@ -17,7 +17,11 @@ export default class ObjectExpression extends Expression {
                 children.skipNonCode();
                 children.assertToken('Punctuator', '}');
             } else {
-                properties.push(children.passNode('Property'));
+                if (children.isNode('Property')) {
+                    properties.push(children.passNode('Property'));
+                } else if (children.isNode('SpreadProperty')) {
+                    properties.push(children.passNode('SpreadProperty'));
+                }
```

Comment:
If node is both not `Property` and `SpreadProperty` we will have something strange.

I suggest rewriting to this:

```js
if (children.isNode('SpreadProperty')) {
    properties.push(children.passNode('SpreadProperty'));
} else {
    properties.push(children.passNode('Property'));
} 
```